flambe/nlp/classification/task.py

```diff
@@ -0,0 +1,204 @@
+from typing import Dict, Optional, Any
+
+import torch.nn as nn
+
+from flambe.dataset import TabularDataset
+from flambe.sampler import BaseSampler
+from flambe.nn import Module, Encoder, Embedder, Embeddings
+from flambe.metric import Accuracy
+from flambe.field import TextField, LabelField
+from flambe.task import Training
+from flambe.optim.optimizer import Optimizer
+from flambe.optim.lr_scheduler import LRScheduler
+
+
+class TextClassifier(Encoder):
```

Comment:
As I said previously, I would put `TextClassifier` in a `model` module.

Reply:
i ll make that change